### PR TITLE
Homebrew: update instructions following 10.11 changes

### PR DESCRIPTION
**Pull request by sbesson**
Documents changes in https://github.com/ome/omero-install/pull/120

### CODE REVIEW

File: omero/sysadmins/unix/server-install-homebrew.txt [hunk 1]
```diff
@@ -333,6 +330,16 @@ try::
     $ brew cleanup
     $ brew link ossp-uuid
 
+For recent versions of OSX, some directories are missing preventing PostgreSQL
```
Comment (mtbc) [inline omero/sysadmins/unix/server-install-homebrew.txt]: Usually we say "OS X" -- probably good to have the docs consistent.

Comment (mtbc) [inline omero/sysadmins/unix/server-install-homebrew.txt]: can we be any more specific about "recent versions"? (if not that's okay)

File: omero/sysadmins/unix/server-install-homebrew.txt [hunk 2]
```diff
@@ -333,6 +330,16 @@ try::
...
+from starting up. In that case, it should be sufficient to reinitialize a
+PostgreSQL database cluster as:
```
Comment (mtbc) [inline omero/sysadmins/unix/server-install-homebrew.txt]: needs `::`

File: omero/sysadmins/unix/server-install-homebrew.txt [hunk 3]
```diff
@@ -333,6 +330,16 @@ try::
     $ brew cleanup
     $ brew link ossp-uuid
 
+For recent versions of OS X (10.10 and above), some directories may be missing
```
Comment (rleigh-codelibre) [inline omero/sysadmins/unix/server-install-homebrew.txt]: The comma here should be after `missing`.

**Comment by rleigh-codelibre** [comment]
Overall looks fine other than the minor punctuation placement noted.

**Comment by rleigh-codelibre** [comment]
Thanks, all looks good to merge to me.

**Pull request merged**